services/ui-src/src/components/fields/DropdownField.tsx

```diff
@@ -92,6 +92,7 @@ export const DropdownField = ({
   return (
     <Box sx={sxOverride} className={`${nestedChildClasses} ${labelClass}`}>
       <CmsdsDropdown
+        data-testid="admin-dropdown"
```

Comment:
Blocking: This adds the test id to all instances of dropdowns in the app, which is probably not what you wanted.